src/nncf/quantization/algorithms/weight_compression/constants.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2025 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+
+from nncf.tensor import Tensor
+from nncf.tensor import TensorDataType
+
+NF4_QUANTILES = np.array(
+    [
+        -1.0,
+        -0.6961928009986877,
+        -0.5250730514526367,
+        -0.39491748809814453,
+        -0.28444138169288635,
+        -0.18477343022823334,
+        -0.09105003625154495,
+        0.0,
+        0.07958029955625534,
+        0.16093020141124725,
+        0.24611230194568634,
+        0.33791524171829224,
+        0.44070982933044434,
+        0.5626170039176941,
+        0.7229568362236023,
+        1.0,
+    ],
+    dtype=np.float32,
+)
+
+
+def get_cb4_quantiles() -> Tensor:
+    """
+    Returns the quantiles for the CB4 codebook.
+    """
+    CB4_QUANTILES = np.array(
```

Comment:
Define global vraible as global or use lower case in function 
https://github.com/openvinotoolkit/nncf/blob/develop/docs/styleguide/PyGuide.md#47-naming